File: app/src/main/java/io/github/vvb2060/keyattestation/AppApplication.java
package io.github.vvb2060.keyattestation;

import android.app.Application;
import android.content.Context;

public class AppApplication {

    public static final String TAG = "KeyAttestation";

    public static Application app;

    public static void init(Application application) {
        app = application;
    }
}

File: app/src/main/java/io/github/vvb2060/keyattestation/AppApplication.kt
package io.github.vvb2060.keyattestation

import android.app.Application
import android.content.Context
import me.bmax.apatch.apApp

/**
 * Bridge class to adapt KeyAttestation code to FolkPatch's Application
 */
object AppApplication {
    const val TAG = "KeyAttestation"
    
    @JvmStatic
    val app: Application
        get() = apApp
    
    @JvmStatic
    val context: Context
        get() = apApp
}
